src/chains/algorand_1/algoErrors.ts

```diff
@@ -14,15 +14,16 @@ export const ChainErrorRegExs: { [key: string]: string } = {
     '(Sig or Msig|mystery sig|LogicSig not enabled|LogicSig.Logic too|LogicSig.Logic version|LogicSig.Logic bad|one of sig or msig)', // the permission isnt valid (or permission already exists in an account)
   AuthUnsatisfied: '(signature validation|multisig validation|signed and not a Logic-only)', // all permission or keys needed for transaction weren't provided
   AuthMissing: '(no sig|LogicSig.Logic empty)', // missing permission or key
-  BlockDoesNotExist: '(no blocks|previous block|block round|block branch|MakeBlock|unrecognized blockhash)',
+  BlockDoesNotExist:
+    '(/ps1/v1/block|no blocks|previous block|block round|block branch|MakeBlock|unrecognized blockhash)',
```

Comment:
'/ps1/v1/block' cant be right. That would imply that the only error thrown by the /block api endpoint is a block not found. This could cause unintended problems with the code as it will keep looking for missing blocks if some other error is thrown. You should never use a phrase in error mapping that could map to multiple errors - except for the 'Misc...' types. Replace this with something in the error message that can only mean a block doesnt exit.